rpc/src/optimistically_confirmed_bank_tracker.rs

```diff
@@ -191,6 +231,27 @@ impl OptimisticallyConfirmedBankTracker {
         }
     }
 
+    fn notify_root_banks(
+        roots: &mut Vec<(Slot, Slot)>,
+        slot_threshold: Slot,
```

Comment:
I feel like this could be `newest_root: &mut Slot` and we just update it to the newest root after `roots.sort_unstable();` below

Reply:
Done